File: app/src/main/java/mishka/rssreader/data/AppDatabase.java
package mishka.rssreader.data;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import mishka.rssreader.data.model.RssItem;
import mishka.rssreader.data.model.RssItemDao;

@Database(entities = {RssItem.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {

    public abstract RssItemDao rssItemDao();
}

File: app/src/main/java/mishka/rssreader/data/AppDatabase.kt
package mishka.rssreader.data;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import mishka.rssreader.data.model.RssItemDao;
import mishka.rssreader.data.model.RssItem

@Database(entities = [ RssItem::class ], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun rssItemDao(): RssItemDao
}
